Update: Read random thoughts from back-end. Web-console error fixed.

jacinda22 opened the pull request:
My portfolio website is now able to display different sentences that I provided in my code to the website. The web console error I was receiving before has also been fixed.

jacinda22 committed:
Update: Read random thoughts from back-end. Web-console error fixed.

perrycate commented on portfolio/src/main/webapp/script.js:
```diff
@@ -26,3 +26,35 @@ function addRandomGreeting() {
+    const responseFromServer = await fetch('/hello');
+    const textFromResponse = await responseFromServer.text();
+
+    const servletContainer = document.getElementById('servlet1-container');
```
Does this actually work? I didn't see an element id with servlet1-container in the html file.

perrycate commented on portfolio/src/main/webapp/script.js:
```diff
@@ -26,3 +26,35 @@ function addRandomGreeting() {
+
+    //parsing to json
+    const ranInfo = await serverResponse.json();
+    //gets a random element w/Math.random(): Error w/ranInfo.length in web console
```
Might want to update this comment since you're no longer getting that error.

perrycate commented on portfolio/src/main/webapp/script.js:
```diff
@@ -26,3 +26,35 @@ function addRandomGreeting() {
+    
+    //printing random message in the website console
+    console.log([Math.floor(Math.random()*ranInfo.length())]);
+    //console.log("info.get(1)");
```
Can delete these commented out lines.

perrycate commented on portfolio/src/main/webapp/script.js:
```diff
@@ -26,3 +26,35 @@ function addRandomGreeting() {
+    const servletContainer = document.getElementById('servlet2-container');
+    servletContainer.innerText = ranElement;
+    //testing what will print to the web console w/ ranInfo.length: prints string instead of int
+    console.log(ranInfo[Math.floor(Math.random()*ranInfo.length)]);
```
If you're done debugging and don't anticipate needing to check these values in the console again, you may want to remove these lines to avoid cluttering up your console output. (It's a very common habit to leave print statements like this in - if you open the console on random websites you'll often see various log messages that were left in by accident.)

jacinda22 committed:
Update/Revision: Deleted excess code.  Contact form was created. Error with connection between form and servlet.

jacinda22 committed:
Merge branch 'main' into Week2-RealPortfolio

jacinda22 committed:
Update Contact Form:
Contact form now asks the user to enter email and choose to request contact information.
